tiatoolbox/annotation/storage.py

```diff
@@ -1117,18 +1118,84 @@ def _load_cases(
         raise IOError("Invalid file handle or path.")
 
     @classmethod
-    def from_geojson(cls, fp: Union[IO, str]) -> "AnnotationStore":
-        geojson = cls._load_cases(
+    def from_geojson(
+        cls,
+        fp: Union[IO, str],
+        scale_factor=1,
+        relative_to=None,
+    ) -> "AnnotationStore":
+        """Create a new database with annotations loaded from a geoJSON file.
+        Args:
+            fp (Union[IO, str, Path]):
```

Comment:
Please can you check if we are using file_path variable in misc? It would be better to be consistent. Also, fp is not clear.

Reply:
I have used fp in everything I have added, to be consistent with what was already used throughout storage.py

Reply:
Have replaced fp with full_path

Reply:
It's supposed to be a common shorthand for file path here (rather than full path). Happy with file_path instead.